orca_detector/web_scraping.py

```diff
@@ -1,4 +1,9 @@
-# script to pull all data from the WHOI website
+"""
```

Comment:
We need to track down the original implementation and give credit here, I've seen it somewhere in GitHub but unfortunately I didn't make a note and cannot find it back ever since

Reply:
I filed to the issue tracker